### ISSUE TITLE
Problem writing accents in Spanish 

### ISSUE THREAD

**Issue by tanrax** [rogual/neovim-dot-app#147]
I wanted to ' Imágenes' . But I added a strange symbol : Im<87>genes.
Not let me write . But I have to copy it from another site. 

![screen shot 2015-09-27 at 21 46 03](https://cloud.githubusercontent.com/assets/4553672/10129740/effffc46-65c1-11e5-9e5e-a519793b2c69.jpg)

**Comment by bambu** [comment]
How are you writing 'Imágenes'? I was not able to reproduce the "Im<87>genes" output that you show in the picture. 

I tried typing it as it would be done with a US keyboard layout on osx (Alt-e a), I also switched to Spanish and Spanish ISO keyboard layouts and in those cases just a regular a would appear due to issue #26.

What did work for me:
- copy and paste the Imágenes
- inserting  á by typing `<c-k>a'` in insertmode

Let me know what you did to get `<87>`

**Comment by tanrax** [comment]
No, if I write the word I appear without the accent. I copy from somewhere else. 

I tried everything you 've told me:

if I write typing <c-k>a' in US keyboard layout: It works well.
if I write typing <c-k>a' in ES keyboard layout: it does not work 
If I copy the word from another program and paste: "Im<87>genes"

thanks to you and the team for this great work :)

**Comment by bambu** [comment]
Can you try the latest? It should allow you to type in ES keyboard. 

As for getting `Im<87>genes` I am not able to reproduce that at all. I have even gone as far as copying all the characters in https://en.wikipedia.org/wiki/List_of_Unicode_characters to Neovim and trying to see if it shows up that way. I have tried copying and pasting from microsoft word as well. In all cases it shows up just fine. 

If you are still getting `im<87>genes` can you tell me a little more about where/how you are copying?

**Comment by tanrax** [comment]
Please watch this video where you can see it in action: https://drive.google.com/file/d/0B8oKsZ4OrGWqMldZajRMaHFtXzA/view?usp=sharing

**Comment by bambu** [comment]
Hmm.. I still can't reproduce it. Sorry. :(  Just out of curiosity, are you using the latest Neovim and nvim? The latest Neovim that was merged last week should take care of the entering the character. 

Here´s what I tried to do to get `<87>`
1. Switching to "Spanish ISO", opening finder, typing mamá, highlighting it, Cmd-C to copy, go to vim and paste it both in insert mode and normal mode. 
2. Checked out different versions of Neovim and nvim build and tried the first step 

In all cases it still works for me.

**Comment by tanrax** [comment]
Yes, I have everything updated . I tried everything you said without success. I not like most I can help you, sorry.
